packages/ckeditor5-show-blocks/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@ckeditor/ckeditor5-show-blocks",
+  "version": "38.0.1",
```

Comment:
It should be `0.0.1` for a new feature :point\_right: https://www.notion.so/Creating-a-new-package-f3528d73f2324413a8222ac4e8ec1334?pvs=4#096cda7ca7344555936ded645b66815e